ocppj/dispatcher.go

```diff
@@ -422,7 +436,9 @@ func (d *DefaultServerDispatcher) CreateClient(clientID string) {
 func (d *DefaultServerDispatcher) DeleteClient(clientID string) {
 	d.queueMap.Remove(clientID)
 	if d.IsRunning() {
```

Comment:
NOTE: this is *still* racy since the channel might have been closed after having checked for `IsRunning`. Should not use this here.